### ISSUE TITLE
The URL in the Web Service Sender formatted sent content is incorrect

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3260]
This was probably just due to a misunderstanding about what "port" means for WSDL documents. It's not a "port" in the traditional TCP sense. Rather it's just a location definition for a concrete binding. The PortType in WSDL 1.1 further muddies the waters; that's just the abstract definition for a service. In fact in WSDL 2.0, PortType has been renamed to Interface and Port has been renamed to Endpoint: [http://www.w3.org/TR/wsdl20-primer/#basics-service]

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3359
Reporter: narupley
Created: 2014-06-30T08:20:33.000-0700

**Comment by rbeckman-nextgen** [comment]
This was fixed along with MIRTH-1790. The formatted sent content no longer includes the port / endpoint in the URL.

Imported Comment. Original Details:
Author: narupley
Created: 2014-07-18T10:44:32.000-0700

**Comment by rbeckman-nextgen** [comment]
Verified the new naming conventions of the sent content by creating two channels in 3.0.3 and processing messages. Exported and imported these channels, sending in new sample data. Finally, I performed a side by side comparison to ensure the correctness of the formatted sent content against an older build and the definition for a wsdl service as linked in the original description.

Imported Comment. Original Details:
Author: jaysenp
Created: 2014-08-27T15:19:30.000-0700
